graphql_compiler/compiler/ir_lowering_sql/__init__.py

```diff
@@ -1,35 +1,148 @@
 # Copyright 2018-present Kensho Technologies, LLC.
 
+import six
+
+from .constants import UNSUPPORTED_META_FIELDS
+from .sql_tree import SqlNode, SqlQueryTree
+from .. import blocks
+from ..ir_lowering_sql.constants import SqlOutput
+from ..metadata import LocationInfo
+from ... import exceptions
+
 ##############
 # Public API #
 ##############
 
+_SKIPPABLE_BLOCKS = (
```

Comment:
nit: These are _types_ of blocks, not blocks themselves -- how about `_SKIPPABLE_BLOCK_TYPES`?